_layouts/default.html

```diff
@@ -2,7 +2,7 @@
 <html lang="en-us">
   <head>
     <meta charset="UTF-8">
-    <title>{{ page.title | default: site.title }}</title>
+    <title>{{ site.title | default: site.github.repository_name }} by {{ site.github.owner_name }}</title>
```

Comment:
Yep. I will just say that this diverges from what the other themes do FYI.

So something like this?

```
<title>{% if page.title %}{{ page.title }} | {% endif %}{{ site.title }}</title>
```